src/ripple/app/paths/README.md

```diff
@@ -1,137 +0,0 @@
-Theory of Pathfinding
```

Comment:
This document still has some kind-of useful information in it? You're right that it's extremely dated though.

Reply:
I went through this and honestly, there's very little captured in that doc that is helpful. I think it's better off removed for now.